ethicml/data/__init__.py

```diff
@@ -1,16 +1,16 @@
 """This module contains items related to data, such as raw csv's and data objects."""
-
-from .adult import *
-from .celeba import *
-from .compas import Compas
-from .configurable_dataset import *
-from .credit import *
-from .dataset import *
-from .genfaces import *
-from .german import *
-from .image_dataset import *
 from .load import *
-from .non_binary_toy import *
-from .sqf import *
-from .toy import *
-from .util import *
+from .tabular_data.adult import *
+from .tabular_data.compas import Compas
+from .tabular_data.configurable_dataset import *
+from .tabular_data.credit import *
+from .tabular_data.dataset import *
+from .tabular_data.german import *
+from .tabular_data.non_binary_toy import *
+from .tabular_data.sqf import *
+from .tabular_data.toy import *
+from .tabular_data.util import *
+from .vision_data.celeba import *
+from .vision_data.cmnist import *
+from .vision_data.genfaces import *
+from .vision_data.image_dataset import *
```

Comment:
why make this unrelated change in this PR?

Reply:
I would also like to know!